npm/webpack-batteries-included-preprocessor/index.js

```diff
@@ -181,11 +181,13 @@ preprocessor.defaultOptions = {
 }
 
 preprocessor.getFullWebpackOptions = (filePath, typescript) => {
-  const options = { typescript }
+  const options = typescript ? { typescript } : {}
 
   options.webpackOptions = getDefaultWebpackOptions()
 
-  addTypeScriptConfig({ filePath }, options)
+  if (options.typescript) {
```

Comment:
so the variable we want to check is typescript.options.typescript? Is there a better name for the top variable? Or could we get options directly passed in?

Reply:
It wasn't quite typescript.options.typescript, but taking another look at this, I realized it had grown a little convoluted so I refactored it a bit in https://github.com/cypress-io/cypress/pull/25931/commits/fa105050e7c25ca05dafa80ebed7d712acf36bcf